gx_suite_definitions/team_info.ipynb

```diff
@@ -0,0 +1,242 @@
+    "validator.expect_column_values_to_not_be_null(\"team\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"team\", min_value=3, max_value=50)\n",
+    "validator.expect_column_values_to_be_in_set(\n",
+    "    \"team\", {\n",
+    "        \"Columbia-Rush\",\n",
+    "        \"Duke\",\n",
+    "        \"Emory\",\n",
+    "        \"Harvard-MIT\",\n",
+    "        \"MSSM - Roussos Lab\",\n",
+    "        \"MSSM - Zhang Lab\",\n",
+    "        \"Sage Bionetworks\",\n",
+    "        \"Mayo-UFL-ISB\",\n",
+    "        \"Chang Lab\", \n",
+    "        \"JAX-VUMC-UW Resilience\",\n",
+    "        \"ASU\",\n",
+    "        \"Longo Lab\",\n",
+    "        \"Duke BARU\", \n",
+    "        \"Emory-Sage-SGC\",\n",
+    "        \"IUSM-Purdue\",\n",
+    "        \"Mayo\"\n",
+    "    }\n",
+    ")\n",
+    "validator.expect_column_values_to_be_unique(\"team\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# team_full\n",
+    "validator.expect_column_values_to_be_of_type(\"team_full\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"team_full\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"team_full\", min_value=10, max_value=100)\n",
+    "validator.expect_column_values_to_match_regex(\"team_full\", \"^[a-zA-Z0-9-() ]+$\")\n",
+    "validator.expect_column_values_to_be_unique(\"team_full\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# program\n",
+    "validator.expect_column_values_to_be_of_type(\"program\", \"str\")\n",
+    "validator.expect_column_values_to_be_in_set(\"program\", {'Community Contributed', None, 'AMP-AD', 'Resilience-AD', 'TREAT-AD'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# description\n",
+    "validator.expect_column_values_to_be_of_type(\"description\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"description\")\n",
+    "validator.expect_column_values_to_match_regex(\"description\", \"^[a-zA-Z0-9-(),.':/ ]+$\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"description\", min_value=10, max_value=2000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# members\n",
+    "validator.expect_column_values_to_be_of_type(\"members\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"members\")\n",
+    "#get JSON schema\n",
+    "with open(\"../src/agoradatatools/great_expectations/gx/json_schemas/team_info/members_schema.json\", \"r\") as file:\n",
+    "    gene_biodomains_schema = json.load(file)\n",
+    "validator.expect_column_values_to_match_json_schema(\"members\", json_schema=gene_biodomains_schema)"
```

Comment:
Rename from `gene_biodomains_schema` -> `members_schema`

Reply:
changed in latest commit